minorminer/utils/parallel_embeddings.py

```diff
@@ -0,0 +1,630 @@
+# Copyright 2024 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Methods provided for the generation of one or more disjoint embeddings. 
+These methods sequentially generate disjoint embeddings of a source graph 
+onto a target graph or provide supporting functionality.
+"""
+import warnings
+
+import dwave_networkx as dnx
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+from typing import Union, Tuple
+
+from minorminer.subgraph import find_subgraph
+
+from minorminer.utils.feasibility import (
+    embedding_feasibility_filter,
+    lattice_size_lower_bound,
+)
+
+
+def visualize_embeddings(
```

Comment:
Should be moved to dwave_networkx with other visualization tools? Other input welcome

Reply:
Maybe, yes? I think it sounds more reasonable to add it to `dwave-networkx` as it's currently organized.

Reply:
What is the difference between this and `dwave_networkx.drawing.qubit_layout.draw_embedding()`? (or the other drawing functions in `dwave-networkx`)?

Reply:
draw_embedding colours by chains. in this method, the embeddings are coloured by subgraph